docs/metrics/index.md

```diff
@@ -127,6 +127,26 @@ biases, and its intended uses.
 
 </div>
 
+## Hallucination
```

Comment:
I think we want to name this `Large Language Model Evaluation`

Reply:
LLM eval is a subset of NLP, right?